Structure_Engine/Create/Constraints/BarRelease.cs

```diff
@@ -60,16 +61,34 @@ public static BarRelease BarReleasePinPin(string name = "PinPin")
 
         /***************************************************/
 
+        [Description("Creates a Bar release that is pinned in one end and axially releaased in the other. \n" +
+                     "This means that the start node will have all tranlational degrees fixed as well as the RX, to avoid instability, i.e. to prevent the Bar from spinning around its own axis. \n" +
+                     "The end node will have the translational y and z directions fixed, all other free, i.e. it will be free to rotate as well as tranlate along the axis of the Bar.")]
```

Comment:
The end node will be fixed in the y-axis and z-axis translationally, all degrees of freedom will be free, i.e. it will be free to rotate as well as translate about the local x-axis of the Bar.